exercises/practice/hamming/HammingTest.php

```diff
@@ -31,45 +9,92 @@ public static function setUpBeforeClass(): void
         require_once 'Hamming.php';
     }
 
-    public function testNoDifferenceBetweenIdenticalStrands(): void
+    /**
+     * uuid: f6dcb64f-03b0-4b60-81b1-3c9dbf47e887
+     * @testdox Empty strands
+     */
+    public function testEmptyStrands(): void
+    {
+        $this->assertEquals(0, distance('', ''));
+    }
+
+    /**
+     * uuid: 54681314-eee2-439a-9db0-b0636c656156
+     * @testdox Single letter identical strands
+     */
+    public function testSingleLetterIdenticalStrands(): void
     {
         $this->assertEquals(0, distance('A', 'A'));
     }
 
-    public function testCompleteHammingDistanceOfForSingleNucleotideStrand(): void
+    /**
+     * uuid: 294479a3-a4c8-478f-8d63-6209815a827b
+     * @testdox Single letter different strands
+     */
+    public function testSingleLetterDifferentStrands(): void
     {
-        $this->assertEquals(1, distance('A', 'G'));
+        $this->assertEquals(1, distance('G', 'T'));
     }
 
-    public function testCompleteHammingDistanceForSmallStrand(): void
+    /**
+     * uuid: 9aed5f34-5693-4344-9b31-40c692fb5592
+     * @testdox Long identical strands
+     */
+    public function testLongIdenticalStrands(): void
     {
-        $this->assertEquals(2, distance('AG', 'CT'));
+        $this->assertEquals(0, distance('GGACTGAAATCTG', 'GGACTGAAATCTG'));
```

Comment:
For better readability in online editor:

```suggestion
        $this->assertEquals(0, distance(
            'GGACTGAAATCTG',
            'GGACTGAAATCTG'
        ));
```